src/python/gudhi/cubical_complex.pyx

```diff
@@ -85,7 +86,7 @@ cdef class CubicalComplex:
         elif ((dimensions is None) and (top_dimensional_cells is None)
             and (perseus_file != '')):
             if os.path.isfile(perseus_file):
-                self.thisptr = new Bitmap_cubical_complex_base_interface(perseus_file.encode('utf-8'))
+                self.thisptr = new Bitmap_cubical_complex_base_interface(str.encode(perseus_file))
```

Comment:
This seems to revert a recent patch by Vincent, did you have a problematic rebase?

Reply:
Oops, no sorry, there is no particular reason as of why I changed it. Probably a bad copy/paste ;-)